test/parallel/test-buffer-arraybuffer.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const Buffer = require('buffer').Buffer;
+const LENGTH = 16;
+
+var ab = new ArrayBuffer(LENGTH);
```

Comment:
FYI these could all be `const` (and the `for (var` could be `for (let`). Not sure what the test style is given that fast const isn't in master yet.